src/Core/Tools/Enums/ReferenceEventSource.cs

```diff
@@ -10,6 +10,6 @@ public enum ReferenceEventSource
     User,
     [EnumMember(Value = "provider")]
     Provider,
-    [EnumMember(Value = "registrationStart")]
-    RegistrationStart,
+    [EnumMember(Value = "registration")]
+    Registration,
```

Comment:
@Mailman14 Any issues with renaming this?

Reply:
@djsmith85 , as info, there have not been any changes to the reference events repo yet to process the new registration reference events (I added the other event). That work is captured [here](https://bitwarden.atlassian.net/browse/PM-9372).